From 8426a3b0d2ff58566215d31aed769eb801826ee6 Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Thu, 13 Jul 2023 09:27:52 +0530
Subject: [PATCH] WA-Increase boot partition size to 41MB.

Boot image size is going out of max size limit.
Hence increasing size 1 more mb.

Tracked-On: NA
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 groups/boot-arch/project-celadon/BoardConfig.mk | 2 +-
 groups/boot-arch/project-celadon/gpt.ini        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/groups/boot-arch/project-celadon/BoardConfig.mk b/groups/boot-arch/project-celadon/BoardConfig.mk
index 587745b..970fd46 100644
--- a/groups/boot-arch/project-celadon/BoardConfig.mk
+++ b/groups/boot-arch/project-celadon/BoardConfig.mk
@@ -26,7 +26,7 @@ TARGET_RECOVERY_PIXEL_FORMAT := "BGRA_8888"
 #
 
 # NOTE: These values must be kept in sync with BOARD_GPT_INI
-BOARD_BOOTIMAGE_PARTITION_SIZE ?= 41943040
+BOARD_BOOTIMAGE_PARTITION_SIZE ?= 42991616
 SYSTEM_PARTITION_SIZE = $(shell echo {{system_partition_size}}*1024*1024 | bc)
 {{^dynamic-partitions}}
 BOARD_SYSTEMIMAGE_PARTITION_SIZE ?= $(SYSTEM_PARTITION_SIZE)
diff --git a/groups/boot-arch/project-celadon/gpt.ini b/groups/boot-arch/project-celadon/gpt.ini
index 3748e95..56c6904 100644
--- a/groups/boot-arch/project-celadon/gpt.ini
+++ b/groups/boot-arch/project-celadon/gpt.ini
@@ -54,7 +54,7 @@ type = fat
 
 [partition.boot]
 label = boot
-len = 40
+len = 41
 type = boot
 {{#slot-ab}}
 has_slot = true
-- 
2.17.1

